engine/src/flutter/impeller/renderer/backend/vulkan/swapchain/ahb/ahb_swapchain_impl_vk.h

```diff
@@ -21,6 +20,22 @@
 
 namespace impeller {
 
+static constexpr const size_t kMaxPendingPresents = 2u;
+
+struct AHBFrameSynchronizerVK {
```

Comment:
Can we just use the same synchronizer util (with some mods if necessary) for both KHR and ABH? The KHR synchronizer was just an implementation detail and I don't think I did a great job documenting it.

Reply:
There is different behavior in how/when we initialize the semaphores, specifically the external ones. I'd like to keep the swapchains as separate as possible until I know this one actually works in prod...